.github/workflows/update-wpt.yml

```
@@ -0,0 +1,62 @@
+name: WPT update
+
+on:
+  schedule:
+    # Run once a week at 12:00 AM UTC on Sunday.
+    - cron: 0 0 * * *
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  wpt-subsystem-update:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        subsystem:
+          - url
+
+    steps:
+      - uses: actions/checkout@v4@<hash>
+        with:
+          persist-credentials: false
+
+      - name: Update WPT for subsystem ${{ matrix.subsystem }}
+        run: |
+          git node wpt update --subsystem=${{ matrix.subsystem }}
+        env:
+          GH_USER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Run WPT tests
+        run: |
+          git node wpt run --subsystem=${{ matrix.subsystem }}
+        env:
+          GH_USER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
      - name: Run WPT tests
        run: |
          git node wpt run --subsystem=${{ matrix.subsystem }}
        env:
          GH_USER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Maybe if they fail, continue with opening the PR, but add the output to the PR body letting maintainers know that the WPT tests failed?

---

thank you I sent the commit for it